.github/contributing.md

```diff
@@ -0,0 +1,52 @@
+# Contributors Guide
+
+Interested in contributing? Awesome! Before you do though, please read our
+[Code of Conduct](https://slackhq.github.io/code-of-conduct). We take it very seriously, and expect that you will as
+well.
+
+There are many ways you can contribute! :heart:
+
+## Bug Reports and Fixes :bug
+
+- If you find a bug, please search for it in the [Issues](https://github.com/slackapi/manifest-schema/issues), and if it isn't already tracked,
+   [create a new issue](https://github.com/slackapi/manifest-schema/issues/new). Fill out the "Bug Report" section of the issue template. Even if an Issue is closed, feel free to comment and add details, it will still
+   be reviewed.
+- Issues that have already been identified as a bug (note: able to reproduce) will be labelled `bug`.
+- If you'd like to submit a fix for a bug, [follow the maintainers guide](./maintainers_guide.md) and mention the Issue number.
+- Include tests that isolate the bug and verifies that it was fixed.
+
+## New Features :bulb
+
+- If you'd like to add new functionality to this project, describe the problem you want to solve in a [new Issue](https://github.com/slackapi/manifest-schema/issues/new).
+- Issues that have been identified as a feature request will be labelled `enhancement`.
+- If you'd like to implement the new feature, please wait for feedback from the project
+   maintainers before spending too much time writing the code. In some cases, `enhancement`s may
+   not align well with the project objectives at the time.
+
+## Tests :mag:, Documentation :books:, Miscellaneous :sparkles
```

Comment:
```suggestion
## Tests :mag:, Documentation :books:, Miscellaneous :sparkles:
```

Reply:
Good catch, I think my linter was configured to prevent [headers that ended with punctuation](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md026) I disabled this rule 💯